tiledb/sm/array_schema/enumeration.cc

```diff
@@ -450,11 +455,11 @@ void Enumeration::generate_value_map() {
 }
 
 void Enumeration::add_value_to_map(std::string_view& sv, uint64_t index) {
-  if (value_map_.find(sv) != value_map_.end()) {
+  const auto res = value_map_.insert(std::make_pair(sv, index));
```

Comment:
By curiosity, did you also try `emplace`? Does it make a perf difference in this case?